src/iris/api.py

```diff
@@ -2269,9 +2360,17 @@ def on_get(self, req, resp, incident_id):
 
             cursor.execute(single_incident_query_tags, incident['id'])
             incident['tags'] = cursor.fetchall()
-            connection.close()
 
             incident['context'] = ujson.loads(incident['context'])
+            # retrieve dynamic_tracking_notification for each incident
+            incident['dynamic_tracking'] = []
```

Comment:
Could this just be assigned to cursor.fetchall() like in the above examples, avoiding the for loop/etc?

Reply:
changed